package.json

```diff
@@ -73,7 +73,7 @@
     "protractor": "~7.0.0",
     "ts-node": "~8.3.0",
     "tslint": "^6.1.3",
-    "typedoc": "^0.20.36",
+    "typedoc": "^0.21.9",
```

Comment:
Is there any reason to not update further? [0.22.0](https://github.com/TypeStrong/typedoc/blob/master/CHANGELOG.md#v0220-2021-09-10) (at least) would get light/dark mode and from a glance of the breaking changes and this repos config _should_ be ok

Reply:
Oh, I just picked the first unvulnerable version, let me patch it to 22 and see if it works

Reply:
Bumped it to 0.22.18, had to make a change in the json config tho, as they introduce breaking changes in minors.